core/src/lib/breg.h

```diff
@@ -68,7 +68,7 @@ class BareosRegex {
   /* private */
   POOLMEM* expr = nullptr;     /**< search epression */
   POOLMEM* subst = nullptr;    /**< substitution */
-  regex_t preg;                /**< regex_t result of regcomp() */
+  regex_t preg = {};           /**< regex_t result of regcomp() */
   regmatch_t regs[BREG_NREGS]; /**< contains match */
```

Comment:
```suggestion
  regmatch_t regs[BREG_NREGS] = {}; /**< contains match */
```

Reply:
Both structs are declared as extern "C" and therefore no default constructor will be generated for them.